store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -301,34 +401,15 @@ func buildResp(chunks []tipb.Chunk, closureExecutor *closureExecutor, ndvs []int
 	}
 	executors := dagReq.Executors
```

Comment:
What happens if `dagReq` using `root_executor` instead of `executors`?

Reply:
then it will use `HandleMPPDAGReq` instead of `handleCopDAGRequest`

Reply:
note, executors are for tikv and root_executor are for tiflash, here we replaced the "mock" tikv part so we only need to handle the executors case.

Reply:
but for mock tiflash tests, if the planner does not choose a mpp plan, I think it will also call `handleCopDAGRequest `?